sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c

```diff
@@ -520,11 +614,83 @@ void stabilization_indi_rate_run(struct FloatRates rate_sp, bool in_flight)
 #endif
   }
 
-  // WLS Control Allocator
-  num_iter =
-    wls_alloc(indi_du, indi_v, du_min, du_max, Bwls, 0, 0, Wv, indi_Wu, du_pref, 10000, 10);
+    // scale such that u is between 0 and 1
```

Comment:
calling the allocation seems to need a lot of preliminary code, can't we just hide this in a function of the library ?